### PR TITLE
Add packing for vscript and packing in  to a VPK instead of the BSP for L4D2\Portal2

### PR DESCRIPTION
**Pull request by GEEKiDoS**
#64 Implement packing vscript

### CODE REVIEW

File: CompilePalX/GameConfiguration/GameConfigurationParser.cs [hunk 1]
```diff
@@ -68,9 +68,10 @@ public static List<GameConfiguration> Parse(string filename)
 								CompilePalLogger.LogLine(GetKey(line) + ":" + GetValue(line));
                                 break;
                             case "Light":
-                                game.VRAD = GetFullPath(GetValue(line), game.BinFolder);
+                                var vrad  = GetFullPath(GetValue(line), game.BinFolder);
```
Comment (Exactol) [inline CompilePalX/GameConfiguration/GameConfigurationParser.cs]: Was this done for a specific reason?

File: CompilePalX/Compilers/BSPPack/Pack.cs [hunk 2]
```diff
@@ -129,27 +145,89 @@ public override void Run(CompileContext context)
...
+
+                    /* CompilePalLogger.LogLine("Copying packed bsp to vmf folder...");
 
-                CompilePalLogger.LogLine("Copying packed bsp to vmf folder...");
+                     if (File.Exists(context.BSPFile))
+                     {
+                         if (File.Exists(context.BSPFile + ".unpacked"))
+                             File.Delete(context.BSPFile + ".unpacked");
 
-                if (File.Exists(context.BSPFile))
```
Comment (Exactol) [inline CompilePalX/Compilers/BSPPack/Pack.cs]: Any reason for disabling this?

Comment (GEEKiDoS) [reply CompilePalX/Compilers/BSPPack/Pack.cs]: For debugging on friend's PC, He got a crash on that line ,forgot to change back, sorry

File: CompilePalX/GameConfiguration/GameConfigurationParser.cs (hunk 1 above)
Comment (GEEKiDoS) [reply CompilePalX/GameConfiguration/GameConfigurationParser.cs]: For debugging on friend's PC, he does not have the Visual Studio so I use dnSpy for debugging instead, but the dnSpy does not showing the game.VRAD so I add a local variable for debugging

File: CompilePalX/Compilers/BSPPack/Pack.cs (hunk 2 above)
Comment (GEEKiDoS) [reply CompilePalX/Compilers/BSPPack/Pack.cs]: Forgot to remove the comment on those code

**Pull request merged**